apps/ledger-live-mobile/src/components/DeviceAction/rendering.tsx

```diff
@@ -441,6 +448,71 @@ export function renderError({
   );
 }
 
+export function renderDeviceNotOnboarded({
+  t,
+  device,
+  navigation,
+}: {
+  t: TFunction;
+  device: Device;
+  // TODO: correctly type the navigation prop here AND in the DeviceAction component
+  navigation: any;
+}) {
+  const navigateToOnboarding = () => {
+    if (device.modelId === "nanoFTS") {
```

Comment:
Technically (although we do it everywhere wrongly) we should use `DeviceModelId.nanoFTS` and not a string

Reply:
Good catch, fixed it.